tests/Selector/NamedSelectorTest.php

```diff
@@ -61,8 +62,6 @@ public function testEscapedSelectors($fixtureFile, $selector, $locator, $expecte
         $escaper = new Escaper();
         $locator = $escaper->escapeLiteral($locator);
 
-        $this->iniSet('error_reporting', -1 & ~E_USER_DEPRECATED);
```

Comment:
this is not needed anymore:

- the deprecation is silenced and so not triggered as exception
- when using the PHPunit bridge, it relies on the legacy group to detect that it is OK to call deprecated APIs